es-util.sh

```diff
@@ -133,14 +135,14 @@ log "Getting bindings for ${SERVICE_NAME}."
 SVC_JSON=$(cf curl \
   "/v2/spaces/$(cat ~/.cf/config.json | jq -rc .SpaceFields.Guid)/service_instances?return_user_provided_service_instances=true&q=name%3A${SERVICE_NAME}&inline-relations-depth=1")
 
-ACCESS_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.access_key | select(. != null)')
-SECRET_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.secret_key | select(. != null)')
-BUCKET=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.bucket | select(. != null)')
+ACCESS_KEY=$(echo $SVC_JSON | jq -r '.resources[].entity.service_bindings[].entity.credentials.access_key | select(. != null)' | head -n1 )
```

Comment:
I believe we don't need the `head` if we just make sure to only use the first of the `service_bindings` by putting a `0` in the brackets - so the jq expression becomes
```
'.resources[].entity.service_bindings[0].entity.credentials.access_key | select(. != null)'
```

(and ditto for the other two variables below)